test/fixtures/settings.yml

```diff
@@ -384,3 +384,8 @@ attribute82:
   category: Setting::Provisioning
   default: 120
   description: 'Time in seconds before SSH provisioning times out'
+attribute82:
+  name: destroy_vm_on_host_delete
+  category: Setting::Provisioning
+  default: false
```

Comment:
Oh why this defaults to false? This is incorrect, it should match reality.

Reply:
leftover from first version, sorry